### PR TITLE
Dynamically replace query/content for FB login

### PR DESCRIPTION
**Pull request by maxxcrawford**
+ ~Dependency PR: #265. The current branch will have to merge into the #265 branch before merging into the full `2.0` branch.~
+ Fixes part of #258 

This pull-request replaces standard implemented FB login and then adds an overlay to it. This replacement is necessary as blocking the Facebook JavaScript SDK completely hides the `.fb-login-button` div. This feature also uses the data-attributes on the div to correctly size/display the replaced element. 

## Testing

Recreation Steps: 
+ [Run this version](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext#Using_web-ext) of the web extension locally
+ Visit the login page example of the [test site](https://maxxcrawford.github.io/facebook-implementation-examples/examples/login/)

## Screenshot

With the FBC enabled: 

image

### CODE REVIEW

**Comment by maxxcrawford** [comment]
@groovecoder @lesleyjanenorton  Ready for review!

File: src/content_script.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
 "use strict";
 
+function detectFacebookLoginButton() {
+  // Test for standard implementation (Example: Facebook Docs)
+  const loginButton = document.querySelectorAll(".fb-login-button");
+  for (let item of loginButton) {
+    const fbcClassResetArr = ["fbc-overlay-small", "fbc-overlay"];
+    let replacementClassArr = ["fbc-loginButton"];
```
Comment (groovecoder) [inline src/content_script.js]: prefer `const`

File: src/content_script.js [hunk 2]
```diff
@@ -44,8 +61,10 @@ browser.runtime.onMessage.addListener(message => {
   console.log("message from background script:", message);
   setTimeout(function () {
```
Comment (groovecoder) [inline src/content_script.js]: `setTimeout( () => {`

File: src/content_script.js (hunk 1 above)
Comment (maxxcrawford) [reply src/content_script.js]: Revised. I left the `for (let item…` stand as is. However, it looks like [it should work](https://stackoverflow.com/questions/31987465/ecmascript-2015-const-in-for-loops) as a `const`. I can replace both this line and the two loops in the `detectFacebookOnPage()`  function.

File: src/content_script.js (hunk 2 above)
Comment (maxxcrawford) [reply src/content_script.js]: Revised!

**Pull request merged**